context-extraction/src/main/java/de/atb/context/persistence/context/ContextRepository.java

```diff
@@ -305,7 +305,7 @@ public synchronized Boolean executeSparqlAskQuery(BusinessCase businessCase, Str
 			throw new IllegalArgumentException("Query may not be empty!");
 		}
 		String finalQuery = prepareSparqlQuery(businessCase, query);
-		DataSource dataset = getDataSource(businessCase);
+        Dataset dataset = getDataSource(businessCase);
```

Comment:
nit: formatting in this class should be improved

Reply:
[x] done